### PR TITLE
Prepare for v1.12 as Istio source is already branched

### PR DESCRIPTION
**Pull request by ericvn**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by rvennam — APPROVED** [review]

File: content/en/docs/reference/commands/operator/index.html [hunk 1]
```diff
@@ -874,7 +874,7 @@ <h2 id="envvars">Environment variables</h2>
 <td><code>UNSAFE_ENABLE_ADMIN_ENDPOINTS</code></td>
 <td>Boolean</td>
 <td><code>false</code></td>
-<td>If this is set to true, dangerous admin endpoints will be exposed on the debug interface. Not recommended for production.</td>
+<td>If this is set to true, dangerous admin endpoins will be exposed on the debug interface. Not recommended for production.</td>
```
Comment (craigbox) [inline content/en/docs/reference/commands/operator/index.html:877]: Are these three typos to be expected?
Do they need to be manually fixed on a new branch if so?

Comment (ericvn) [reply content/en/docs/reference/commands/operator/index.html:877]: I made a comment to cherry-pick the fix into 1.12.

**Comment by ericvn** [comment]
Looks like helm failed to install. Running again..

**Comment by ericvn** [comment]
Circuit breaker flake, so running that again.

**Comment by ericvn** [comment]
Helm was failing as it has a validatingwebhook from a prior test. We don't check for that in the before/after so not sure what test is leaving it around. For now, just removing as part of the setup for this test.

**Comment by ericvn** [comment]
Let this merge if the tests pass. We will pick up the spelling update upstream.

**Comment by craigbox** [comment]
ack: only added it as I saw Ram had already approved.

**Pull request merged**